package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "snyk-docker-plugin",
```

Comment:
this doesn't look right, the `packageurl-js` lib did not change

Reply:
I believe that's because the version was already 1.2.0